README.md

```diff
@@ -1,20 +1,35 @@
 [![Build Status](https://travis-ci.org/repolicy/csp-guru.svg?branch=master)](https://travis-ci.org/repolicy/csp-guru)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1318151.svg)](https://doi.org/10.5281/zenodo.1318151)
+# openCSP
+A database of concentrating solar power plants of the world for energy modellers and analysts.
 
-# Welcome to csp.guru!
+## About
+This dataset holds technical, economic, financial, and industrial data on all operating concentrating solar power stations and of stations under construction, as well as several pilot stations. For detailed description of the data collection procedure, please see the initial article for which the dataset was compiled [Lilliestam et al. (2017)](https://doi.org/10.1038/nenergy.2017.94).
```

Comment:
I propose to reduce this README to the absolute minimum, see #57. Otherwise we have to maintain redundant information which is annoying. This README is anyway not super important.

Reply:
Let's do this in a separate PR.

Reply:
you decide!